### ISSUE TITLE
having error displayed afeter the update

### ISSUE THREAD

**Issue by lionelB** [Microsoft/vscode-tslint#87]
hello,

I've just update my tslint plugin, and now, I encountered some error
this message is displayed in toast and is also present in the dev-tools console

```
messageService.ts:119vscode-tslint: 'Cannot read property 'Warning' of undefined' while validating: /Users/lionel/Code/app/service.ts stacktrace: TypeError: Cannot read property 'Warning' of undefined
    at makeDiagnostic (/Users/lionel/.vscode/extensions/eg2.tslint-0.5.35/server/server.js:44:44)
    at /Users/lionel/.vscode/extensions/eg2.tslint-0.5.35/server/server.js:213:30
    at Array.forEach (native)
    at doValidate (/Users/lionel/.vscode/extensions/eg2.tslint-0.5.35/server/server.js:212:22)
    at validateTextDocument (/Users/lionel/.vscode/extensions/eg2.tslint-0.5.35/server/server.js:143:27)
    at Delayer.task (/Users/lionel/.vscode/extensions/eg2.tslint-0.5.35/server/server.js:267:9)
    at /Users/lionel/.vscode/extensions/eg2.tslint-0.5.35/server/delayer.js:21:36
```

Do you know if  there a way to revert to a previous plugin version in visual studio

**Comment by lionelB** [comment]
the weird thing is that tslint works like a charms using the cli

**Comment by lionelB** [comment]
oki so another update... seems to be fine

**Comment by lionelB** [comment]
thanks a lot

**Comment by egamma** [comment]
@lionelB sorry about that. I published a version with some out dated modules. Fixed in 0.5.37.
